runtime/autoload/health/provider.vim

```diff
@@ -268,7 +268,8 @@ function! s:check_python(version) abort
   let python_multiple = []
 
   if exists(loaded_var) && !exists('*provider#'.pyname.'#Call')
-    call health#report_info('Disabled ('.loaded_var.'='.eval(loaded_var).').  This might be due to some previous error.')
+    call health#report_info('Disabled ('.loaded_var.'='.eval(loaded_var).').')
+    return
   endif
```

Comment:
Let's only do this if the variable is `0`.

```
  if exists(loaded_var) && 0 == +eval(loaded_var) && !exists('*provider#'.pyname.'#Call')
    call health#report_info('Disabled ('.loaded_var.'='.eval(loaded_var).').')
    return
  elseif exists(loaded_var) && !exists('*provider#'.pyname.'#Call')
    call health#report_info('Disabled ('.loaded_var.'='.eval(loaded_var).').  This might be due to some previous error.')
  endif
```